### ISSUE TITLE
Folding icons in the panel does not increase with dpi

### ISSUE THREAD

**Issue by zhhzhfya** [JAM-Software/Virtual-TreeView#1268]
Version: Virtual-TreeView-8.0.4
Delphi 11.0
i find the folding icons in the panel does not increase with dpi
and the project Options>Manifest>DPI Awareness>Per Monitor v2

thanks 
<img width="312" alt="Snipaste_2024-10-10_10-44-01" src="https://github.com/user-attachments/assets/a3fef175-ad4d-4c26-8c0e-7455e72b3833">

**Comment by joachimmarder** [comment]
If the screenshots are from a small test project, then please attach it to this issue.

I just modified the included demo project "Advanced" so that the page "Paint line in Windows XP" shows a Virtual Tree on a panel. Could you please download it and check if you can repro the problem with this project.

I was not yet able to repro this, but I am using Delphi 12.

**Comment by zhhzhfya** [comment]
you are right ，project Options>Manifest>DPI Awareness>Per Monitor v2，delphi 12 is ok

**Comment by joachimmarder** [comment]
@zhhzhfya: Does this mean switching DPI-Awareness to "Per Monitor v2" solved your problem?

**Comment by joachimmarder** [comment]
No clear repro steps, so I am closing this issue.
